maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -679,6 +679,16 @@ public abstract class AbstractSurefireMojo extends AbstractMojo implements Suref
     @Parameter(property = "enableAssertions", defaultValue = "true")
     private boolean enableAssertions;
 
+    /**
+     * Flag for including/excluding system-out and system-err elements in xml reports.
+     * <br>
+     * True by default.
+     *
+     * @since 3.0.0
+     */
+    @Parameter(property = "enableOutputElements", defaultValue = "true")
+    private boolean enableOutputElements;
```

Comment:
Please rename to `enableOutErrElements` this will better reflect the intent w/o even reading the docs.